cypress/support/fragments/marcAuthority/marcAuthority.js

```diff
@@ -138,6 +158,9 @@ export default {
   clicksaveAndCloseButton: () => {
     cy.do(saveAndCloseButton.click());
   },
+  clickUnlinkButton: () => {
```

Comment:
Create quickmarc editing methods in quickMarcEditor.js file, but before check there please there are methods already created for this type of actions.

Reply:
fixed